core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -35,6 +36,29 @@
  */
 @Recorder
 public class LoggingSetupRecorder {
+
+    static final boolean IS_WINDOWS = System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("win");
```

Comment:
Just for the sake of avoiding possible NPE, it would be nice to use `getProperty("os.name", "Linux")`, WDYT?

Reply:
if that happens you are on a non-java vm :)